web-ui/src/main/resources/catalog/components/edit/validationreport/partials/validationreport.html

```diff
@@ -45,9 +45,11 @@
            data-ng-class="'schematron-result-list-' + labelImportanceClass(type)">
         <div data-gn-slide-toggle="" class="panel-heading clearfix">
           <div class="col-md-9">
-            <h2 class="gn-schematron-title">{{(type.label || type.id) | translate}}</h2>
+            <h2 class="gn-schematron-title"><span data-ng-if="type.schematronVerificationError">
+              <i class="fa fa-exclamation-triangle text-danger"></i>&nbsp;</span>{{(type.label || type.id) | translate}}</h2>
```

Comment:
I guess this is my only remark, I would rather see a class `fa-fw` than a `&nbsp;`

Reply:
There are more `&nbsp;` in that template. Do I change them too?